CHANGELOG.asciidoc

```diff
@@ -60,6 +60,10 @@ https://github.com/elastic/beats/compare/v8.15.2\...v8.15.3[View commits]
 
 - Memory usage is not correctly limited by the number of events actively in the memory queue, but rather the maximum size of the memory queue regardless of usage. {issue}41355[41355]
 
+*Affecting Metricbeat*
```

Comment:
I think this applies to every 8.15.x release, so you'd have to copy it into the section for each of them.

Reply:
Thanks! Done.
I'm doing the PR against `8.15` branch... 
For 8.16.0, should I redo the same PR or it can be automatically ported there?

How can we force the re-generation of the RN once done?

Reply:
I have always thought just merging PRs to the CHANGELOG.asciidoc would be enough.

This PR would have to be recreated against the 8.16 branch because the 8.16 release notes sections don't exist in the 8.15 branch.